src/server/session.ts

```diff
@@ -120,6 +120,117 @@ namespace ts.server {
         export const Unknown = "unknown";
     }
 
+    namespace Metrics {
+        let eventCounts: Map<number> = {};
+        let properties: Map<string> = {};

```

Comment:
this should be `dtsMaps` instead. and it should be a `FileMap` and not just a `Map`